### DISCUSSION TITLE
Possibility to have Input changes without creating drafts -the inputs that don't affect the element itself-

### DISCUSSION THREAD

**Discussion by vnali** [craftcms/cms#12771] [Ideas] [answered]
Hi!
I have a form on my custom element created in statusFieldHtml(), which has a lightswitch input field. This input doesn't change the status of the current element at all. I only use its value only for passing its status to my desired controller/action. this controller action process passed elementId, and light switch input with some API, saves this element and returns to the element edit page again.

My problem is when I change this input light switch on edit page, Craft thinks that this is a form element field and it creates a provisional draft.

and what is the problem with this provisional draft? because of this provisional draft, I can't see the latest element data on the return to edit page.
I find [this question on SE](https://craftcms.stackexchange.com/questions/40299) which has the same issue that I have with a good explanation.

I am asking if there is a way we can create/mark these inputs on the element edit form so that changing them doesn't trigger draft creation.

**Comment by brandonkelly** [comment, marked as answer]
If you remove the input’s `name` attribute, it won’t trigger autosaving, as it won’t technically be a form input.
